### ISSUE TITLE
// ** Version 2.9.1 Bug ** //

### ISSUE THREAD

**Issue by mattgilroy** [elementor/elementor#10543]
// ** Version 2.9.1 Bug ** //

Media Carousel Widget

On mobile the minimum view is stuck at 3 - when 1 is selected it still shows 3 slides.

Using cover flow and carousel.

## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by fabianenzensperger** [comment]
+1

**Comment by shilo-ey** [comment]
I merged this issue with #10540, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
